src/app/models/abstract.model.ts

```diff
@@ -482,7 +482,15 @@ abstract class AbstractModel<K extends IModelAttributes, J extends IAbstractMode
         returnValue = value.toString();
         break;
       case IModelFillAblesTypes.BOOL:
-        returnValue = !!value;
+          const boolMap = {
```

Comment:
If the api returns null or empty because the backend model attribute is nullable, there will be an error.